utils/typings/styled.d.ts

```diff
@@ -10,7 +10,6 @@ import 'styled-components';
 declare module 'styled-components' {
   export type Hue = Record<number | string, string> | string;
 
-  /* eslint-disable-next-line @typescript-eslint/interface-name-prefix */
```

Comment:
Removing this rule as it has been deprecated in [`v3.0.0`](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0).